common/models/update_load_balancer_request.go

```diff
@@ -1,20 +1,28 @@
 package models
 
 import (
-	"fmt"
-
 	swagger "github.com/zpatrick/go-plugin-swagger"
 )
 
 type UpdateLoadBalancerRequest struct {
-	LoadBalancerID string
-	Ports          *[]Port     `json:"ports"`
-	HealthCheck    HealthCheck `json:"health_check"`
+	LoadBalancerID string       `json:""`
```

Comment:
`json:"load_balancer_id"` (didn't mean `json:""` literal :smile: )

Reply:
Ah I see, ok, this makes more sense.